framework/doc/content/source/auxkernels/FunctorMatPropElementalAux.md

```diff
@@ -0,0 +1,14 @@
+# FunctorMatPropElementalAux
+
+!syntax description /AuxKernels/FunctorMatPropElementalAux
+
+## Overview
+
+This object populates an elemental auxiliary variable by taking the cell-average
+of an AD functor material property.
```

Comment:
this is evaluating at the cell centroid not taking the cell average

Reply:
well actually it's doing whatever was defined for the functor(elem) of this material property. So could be either

Reply:
For a finite volume calculation my statement is true, but your statement is more generally true

Reply:
I edited this statement to be more general